cockroachcloud/private-clusters.md

```diff
@@ -18,12 +14,11 @@ On the other hand, a private {{ site.data.products.dedicated }} cluster's nodes
 
 ## Create a private cluster
 
-To be enrolled in the preview and deploy a new private cluster on AWS or GCP, contact your account team.
+- By default, newly-created {{ site.data.products.dedicated }} clusters deployed on GCP will be private clusters.
+- By default, newly-created {{ site.data.products.dedicated }} clusters deployed on AWS will **not** be private clusters.
 
-After your organization is enrolled in the preview:
+  To create a private cluster on AWS, you must use [{{ site.data.products.db }} API](cloud-api.html) or Terraform Provider(https://registry.terraform.io/providers/cockroachdb/cockroach/latest). When you [create the cluster](https://cockroachlabs.com/docs/api/cloud/v1/clusters), set the `network_visibility` field to `NETWORK_VISIBILITY_PRIVATE`. If you use [CockroachDB's Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/cluster), set `private_network_visibility` to `true`, which in turn sets `network_visibility` to `NETWORK_VISIBILITY_PRIVATE`.
 
-- By default, newly-created {{ site.data.products.dedicated }} clusters deployed on GCP will be private clusters.
-- By default, newly-created {{ site.data.products.dedicated }} clusters deployed on AWS will **not** be private clusters. To create a private cluster on AWS, you can specify a special field using the [{{ site.data.products.db }} API](/docs/cockroachcloud/cloud-api.html). Contact your account team for details.
 - An existing cluster can't be migrated in-place to a private cluster.
```

Comment:
The structure of this section is a little bit difficult to read. It seems like these are mostly considerations when creating a private cluster, and the actual instructions are buried between bullet points. Maybe the first half could be two short paragraphs for GCP and AWS, and the second half after "When you create a private cluster:.." could be a bulleted list, not numbered steps. Just a suggestion

Reply:
Updated this section a bit. Thanks.